### DISCUSSION TITLE
How to set Windows Update policies?

### DISCUSSION THREAD

**Discussion by Callahan78** [amidaware/tacticalrmm#1626] [Q&A] [answered]
I would like to do the windows patch management (approval, install time, reboot policy) completely by Tactical-RMM.
But how do I optimally configure the policies on Windows itself (be it local, be it per GPO), so that nothing gets in each other's way or blocks each other?
Are there any best practices?

**Comment by silversword411** [comment, marked as answer]
You don't have AD?

Read the docs and just set it
https://docs.tacticalrmm.com/howitallworks/#tldr-tactical-rmm-based-patching-recommendation

**Reply by Callahan78** [reply]
I seem to have overlooked that somehow. I will have a look at that. Thanks for the hint.
